client/app/controllers/projects.js

```diff
@@ -1,41 +1,84 @@
 import Controller from '@ember/controller';
+import { sort } from '@ember/object/computed';
+
+// TODO: Get theses codes from a Constants service
+const PACKAGE_STATUS_CODES = {
+  PACKAGE_PREPARATION: 'Package Preparation',
+  CERTIFIED: 'Certified',
+  FINAL_APPROVAL: 'Final Approval',
+  REVIEWED_NO_REVISIONS_REQUIRED: 'Reviewed - No Revisions Required',
+  REVIEWED_REVISIONS_REQUIRED: 'Reviewed - Revisions Required',
+  WITHDRAWN: 'Withdrawn',
+  SUBMITTED: 'Submitted',
+  UNDER_REVIEW: 'Under Review',
+  MISTAKE: 'Mistake',
+};
+
+const PACKAGE_VISIBILITY_CODES = {
+  APPLICANT_ONLY: 717170002,
+  GENERAL_PUBLIC: 717170003,
+};
 
 export default class ProjectsController extends Controller {
   queryParams = ['email'];
 
   // TODO: organize this business logic as computed properties on the projects model
```

Comment:
I would emphasize this TODO here for future reference. These computeds here don't depend on any other context outside of the model itself, so could probably just live there.